ad_hoc/import_eoni.yaml

```diff
@@ -0,0 +1,30 @@
+---
+- hosts: all
+  vars_files:
+    - "../vars.yml"
+  gather_facts: true
+  become: true
+  become_user: "{{ project_name }}"
+
+  tasks:
+
+  - name: Pull sources from the repository.
+    git:
+      repo: "{{ project_repo }}"
+      dest: "{{ project_root }}/code/"
+      version: "{{ branch }}"
+      accept_hostkey: true
+      force: yes
+    notify:
+      - restart web frontend
+
+  - name: Flush handlers
+    meta: flush_handlers
+
+  - name: Import EONI from s3
+    shell: "~/import_eoni_from_s3.sh"
+    args:
+      chdir: "{{ project_root }}/code/"
+
+  handlers:
+    - import_tasks: "../handlers.yml"
```

Comment:
@symroe would be good to know if I've done anything obviously silly here. Otherwise I'll rebase and merge.